lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dvla/endpoints/DriverMatchService.java

```diff
@@ -52,6 +53,8 @@ public class DriverMatchService {
 
     private static final String DVLA_RESPONSE_HEADER_REQUEST_ID_KEY = "X-DVLA-Request-Id";
 
+    private static final String HTTP_404_EXPECTED_ERROR_CODE = "ENQ018";
```

Comment:
Are you sure its always going to be this code can we not just trust 404 on this endpoint will be not found